### ISSUE TITLE
SOTA for predictors

### ISSUE THREAD

**Issue by zacharopoulou** [BioGeMT/FuNmiRBench#42]
### This issue will define the total of predictors used for evaluation

**Scope:**

- Standardized annotation mapping
- Per-algorithm sanity checks
- Score harmonization
- Predictor classification

**Standard Annotation (Mandatory for All Algorithms)**

- All algorithm outputs must be mapped to: Ensembl v115 | miRBase v22.1

- Required output fields (mandatory schema):
          Ensembl_ID (e.g., ENSG00000123456)
          Gene_Name (e.g., MARK1)
          miRNA_ID (e.g., MIMAT0000062)
          miRNA_Name (e.g., hsa-let-7a-3p)
          score (normalized to range 0-1 with 1 be most confident interaction - always 3 digits after delimeter)

Every wrapper must normalize outputs into this exact schema.

**Sanity Checks Per Algorithm**
(1) Training Data Leakage - Were RNA-Seq experiments used during training? If yes, those experiments must be excluded from our benchmark.

(2) Gene-Level Score Availability, A gene-level score is required. How the gene-level score was calculated? By mean, max from MRE level scores? By a meta-classifier? Algorithms that included a separate methodology for Gene-level score only must be included

(3) Score Semantics & Coverage. For each algorithm document: Score type (probability, regression etc..) | Score range (e.g. [0–1], unbounded, etc.) | Recommended threshold (if provided) | Do they output all evaluated pairs? Only positives? | Define how missing pairs will be treated during evaluation.

(4) Predictor Type Classification
Classify each algorithm as:

NOTE: Standalone predictor | Meta-predictor? (aggregates other tools) 

**Deliverables Per Algorithm:**

normalize_<algorithm>.py

Final Goals:

- Specify ground truth tsv file for predictors
- Define the total of predictors that the pipeline will use for evaluation
- Examine final overlaps of mirnas and genes | check RNA-Seq if we need to exclude/include more data
